org/wildfly/undertow/host/capability.adoc

```diff
@@ -0,0 +1,31 @@
+NAME: org.wildfly.undertow.host
+
+DESCRIPTION: Undertow web server host interface, main entry point for applications & context to be bound to
+
+REGISTERED BY:
+
+  NAME: WildFly
+  URL:  https://github.com/wildfly/wildfly
+
+DYNAMIC: true
+
+PRIVATE: false
```

Comment:
Same comment here as for the UndertowService capability.

One general point:

public class Foo implements Service<Foo>

makes Foo wrong as a SERVICE PROVIDED class. The Service API should not be part of the contract; outside users shouldn't be able to invoke start/stop/getValue. Those are for MSC.